pkg/console/subresource/deployment/deployment.go

```diff
@@ -162,9 +167,9 @@ func DefaultDeployment(operatorConfig *operatorv1.Console, cm *corev1.ConfigMap,
 					TerminationGracePeriodSeconds: &gracePeriod,
 					SecurityContext:               &corev1.PodSecurityContext{},
 					Containers: []corev1.Container{
-						consoleContainer(operatorConfig),
+						consoleContainer(operatorConfig, workingVolumeConfigList),
```

Comment:
We need to verify that changes here cause new pods to roll out.  I would assume so, but as I'm tinkering with this, I don't see it happening.  

We might have to pull the `meta.Annotations[customLogoFile] = <hackery>` trick if its not automatic.

Reply:
Changes to the pod template trigger a rollout.

Reply:
Right, it should work.  But when I was running the code, I had to manually delete pods to see updates.  So something isn't quite wired up correctly.

Reply:
Also, we should ensure this works:

```bash
// user gives us a logo
oc create configmap my-logo-file --from-file ~/Desktop/fake-logo.png -n openshift-console
oc create configmap my-logo-file --from-file ~/Desktop/another/dirfake-logo.png -n openshift-console
```
Either way, the configmap will generate:

```
binaryData:
  fake-logo.png: iVBORw0KGgoAAA.....5ErkJggg==
```
The `fake-logo.png` will be the same, but the image will be different.  When we create the volume mount this will probably not trigger a pod rollout.

Reply:
Correct, changing the key or image file will not change the deployment and therefore not force a rollout of the console pods. Changing the configmap name or simply deleting the deployment does work, but that would be a manual step for the user.

Reply:
We shouldn't need to trigger a rollout in the case when only the image itself changes. The mounted config map will update in the pod, and we server the logo file directly from the filesystem.

Reply:
I was able to verify (by manually moving the config map to the openshift-console ns for now) that changing the image in the configmap will show up in the pod and then on the console UI.

Reply:
The configmap now gets copied automatically using an `informer` to trigger the update.

Reply:
Code now uses a `ResourceSyncer` instead of an `informer`